addons/hr_expense/wizard/hr_expense_split_wizard_views.xml

```diff
@@ -40,7 +40,8 @@
                     </group>
                     <field name="split_possible" invisible="1"/>
                     <footer>
-                        <button name="action_split_expense" string="Split Expense" type="object" class="oe_highlight"  data-hotkey="q"/>
+                        <button name="action_split_expense" invisible="split_possible" string="Split Expense" type="object" class="oe_highlight disabled"  data-hotkey="q"/>
```

Comment:
```suggestion
                        <button name="action_split_expense" invisible="split_possible" string="Split Expense" type="object" class="oe_highlight disabled"/>
```
Don't need data hotkey since the button is disabled

Reply:
@mega-odoo 
You should need to remove `data-hotkey="q"` from 16.0; it will generate an error when users press ALT + Q with the disabled button.

Reply:
> @mega-odoo You should need to remove `data-hotkey="q"` from 16.0; it will generate an error when users press ALT + Q with the disabled button.

This issue will be resolved in this PR: https://github.com/odoo/odoo/pull/190323 Thank you :)